baselines/ppo2/runner.py

```diff
@@ -63,8 +89,10 @@ def other_agent_action():
 
                     # If there are environments selected to not run in SP, generate actions
                     # for the other agent, otherwise we skip this step.
+                    #TODO: It's inefficient to calculate all actions, even though only 1 might be used?
```

Comment:
Yes this is right. However, the overhead for computing n vs 1 action when using a model is actually quite small when compared to 0 vs 1. This is why I ended up computing it twice (the code was also getting already quite messy, and I wanted to reduce the chance of bugs). We might however want to try and fix this issue in the future.